### PR TITLE
chore: update CHANGELOG

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR updates the `CHANGELOG.md` by adding #3330

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
+  - [3.11.0](#3110)
     - [New Features](#new-features)
     - [Bug Fixes](#bug-fixes)
   - [Documentation](#documentation)
   - [Development](#development)
+  - [3.10.0](#3100)
+    - [New Features](#new-features-1)
+    - [Bug Fixes](#bug-fixes-1)
+  - [Documentation](#documentation-1)
+  - [Development](#development-1)
```
Comment (oxy) [inline CHANGELOG.md:15]: I sooo didn't catch this last time around - the indentation is bad :sweat_smile:

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,6 +52,26 @@ VS Code v0.00.0
...
+
+### Bug Fixes
+
+- fix: Check the logged user instead of $USER #3330 @videlanicolas
+
+## Documentation
+
+- item
+
+## Development
```
Comment (oxy) [inline CHANGELOG.md:71]: `##`  -> `###`

**Review by oxy — CHANGES_REQUESTED** [review]
I think we can also fix the indentation here while we're at it! I can't believe I didn't catch this in review earlier >.<

File: CHANGELOG.md (hunk 2 above)
Comment (jsjoeio) [reply CHANGELOG.md:71]: Nice catch!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
 # Changelog
 
 - [Changelog](#changelog)
+- [Changelog](#changelog-1)
```
Comment (oxy) [inline CHANGELOG.md]: Oh no, what happened here :fearful: 
I think there should be only one? I'm confused.

**Review by oxy — CHANGES_REQUESTED** [review]

**Pull request merged**